packages/cli/src/commands/deploy/args.ts

```diff
@@ -7,6 +7,7 @@ export const help = () => {
   ${chalk.bold(`${logo} ${getPkgName()} [deploy]`)} [path-to-project] [options]
 
   --prod                         Create a production deployment
+  --skip-build, --prebuilt       Use prebuilt output from ".vercel/output"
```

Comment:
We could alternatively add the alias, but leave `--prebuilt` as the documented flag for now.